frontend/containers/discover-map/map-layers-selector/component.tsx

```diff
@@ -62,116 +50,104 @@ export const MapLayersSelector: FC<MapLayersSelectorProps> = ({
-                        {layerGroup.layers.map(
-                          ({
-                            id,
-                            name,
-                            description,
-                            overview,
-                            dataSource,
-                            dataSourceUrl,
-                            group,
-                          }) => (
-                            <li key={id} className="flex items-center gap-1.5">
-                              <label
-                                key={id}
-                                htmlFor={id}
-                                className="flex items-center gap-2 cursor-pointer"
-                              >
-                                <Switch
-                                  id={id}
-                                  name={group}
-                                  defaultValue={id}
-                                  switchSize="smallest"
-                                  register={register}
-                                  registerOptions={registerOptions}
-                                />
-                                {name}
-                              </label>
-
-                              <Button
-                                theme="naked"
-                                size="smallest"
-                                className="flex items-center justify-center w-4 h-4 text-gray-800 scale-90 border border-gray-800 rounded-full pointer focus-visible:outline-green-dark focus-visible:outline-2"
-                                onClick={() =>
-                                  handleShowTooltip({
-                                    id,
-                                    name,
-                                    description,
-                                    overview,
-                                    dataSource,
-                                    dataSourceUrl,
-                                  })
-                                }
-                              >
-                                <span className="sr-only">
-                                  <FormattedMessage defaultMessage="Information" id="E80WrK" />
-                                </span>
-                                <span className="pt-0.5 text-xs" aria-hidden>
-                                  i
-                                </span>
-                              </Button>
-                            </li>
-                          )
-                        )}
-                      </ol>
-                    </Expando>
-                  );
-                })}
-              </form>
-            </div>
-            <LayerTooltip
-              selectedLayer={selectedLayer}
-              closeTooltip={() => setSelectedlayer(undefined)}
-            />
+            {visibleLayers}
+          </div>
+        )}
+      </button>
+      <FocusScope contain restoreFocus>
```

Comment:
Before you made changes to this file, the `FocusScope` was only rendered when the menu was open. Now with your changes, it is always rendered. As a consequence, the user can not navigate Discover with the keyboard anymore because the focus is trapped in the menu (which appears closed).

You could make this change to fix the problem:
```suggestion
      <FocusScope contain={layerSelectorOpen} restoreFocus>
```